simulator/src/main/cpp/ChirpBluetoothRxTx.h

```diff
@@ -2,6 +2,6 @@
 class ChirpBluetoothRxTx
 {
     public:
-        virtual void send(char[], size_t) = 0;
-        virtual char* receive() = 0;
+        virtual void send(char[], size_t);
```

Comment:
What is the reason for this not being pure virtual? I assume that there should never be a ChirpBluetoothRxTx object, as its just an interface?

Reply:
made virtual again. forgot to revert after testing some inheritance stuff.